pkg/core/pipeline/autodiscovery/config/main.go

```diff
@@ -22,8 +22,8 @@ type Input struct {
 	ScmSpec *scm.Config
 	// ScmID defines the scmid associated to the scm specification
 	ScmID string
-	// PullRequestSpecSpec defines the pullrequest specificiation
-	PullRequestSpec *pullrequest.Config
-	// ScmID defines the scmid associated to the scm specification
-	PullrequestID string
+	// ActionSpec defines the action specification
+	ActionSpec *action.Config
+	// ActionID defines the scmid associated to the scm specification
+	ActionID string
```

Comment:
Same question as https://github.com/updatecli/updatecli/pull/990/files#r1027279967 @olblak .

I'm not sure where/how is this struct is used though so not sure if it user visible? (https://www.updatecli.io/docs/core/autodiscovery/)

Reply:
This is the default pullrequest settings assigned to newly generated manifest.
Such as https://github.com/updatecli/updatecli/blob/main/e2e/updatecli.d/warning.d/autodiscovery/fleet/githubPullrequest.yaml#L12

Reply:
🤔 Mmmh I'm not sure to understand to what is mapped, in the YAML, the `Input` struct?

It sounds like an "internal" object passed to the `DiscoverManifests()` method so not sure if it is exposed to the end user?

I'm asking because if it is an internal object no need to add a depreciation message (compared to the `Config` struct above).

Reply:
You are right the the input struct is internal only